osidb/tests/test_filters.py

```diff
@@ -0,0 +1,451 @@
+        """Test that get_filters() properly adds 'in' filters"""
+        # Get the filters from AffectFilter
+        filters = AffectFilter.get_filters()
+
+        # Check that regular fields have both exact and __in versions
+        assert "ps_module" in filters  # base filter
+        assert "ps_module__in" in filters  # auto-added 'in' filter
+
+        assert "ps_component" in filters
+        assert "ps_component__in" in filters
+
+        assert "affectedness" in filters
+        assert "affectedness__in" in filters
+
+        # Check that datetime fields with comparison lookups don't get __in
+        assert "created_dt" in filters
+        assert "created_dt__gt" in filters
+        assert "created_dt__gte" in filters
+        # created_dt__in should NOT be auto-added since it has gt/gte lookups
+        assert "created_dt__in" not in filters
+
+    def test_in_filter_respects_existing_in_filters(self):
+        """Test that auto-in doesn't duplicate if 'in' is already in the lookup list"""
+        # FlawFilter already has explicit __in filters, ensure no conflicts
+        filters = FlawFilter.get_filters()
+
+        # These should exist and not cause issues
+        assert "cve_id" in filters
+        assert "impact" in filters
+        assert "source" in filters
+
+    def test_in_filter_with_uuid_fields(self, auth_client, test_api_v2_uri):
+        """Test that UUID fields work with auto-in filter"""
+        flaw1 = FlawFactory(embargoed=False)
+        flaw2 = FlawFactory(embargoed=False)
+        affect1 = AffectFactory(flaw=flaw1)
+        affect2 = AffectFactory(flaw=flaw2)
+
+        # Test filtering by single UUID first
+        response = auth_client().get(f"{test_api_v2_uri}/affects?uuid={affect1.uuid}")
+        assert response.status_code == status.HTTP_200_OK
+        assert response.json()["count"] == 1
+
+        # Test filtering by multiple UUIDs with __in suffix
+        response = auth_client().get(
+            f"{test_api_v2_uri}/affects?uuid__in={affect1.uuid},{affect2.uuid}"
+        )
+        assert response.status_code == status.HTTP_200_OK
+        results = response.json()["results"]
+        assert len(results) == 2
+        uuids = {r["uuid"] for r in results}
+        assert uuids == {str(affect1.uuid), str(affect2.uuid)}
+
+    def test_in_filter_for_nested_fields(self, auth_client, test_api_v2_uri):
+        """Test that auto-in works for nested relationship fields"""
+        flaw1 = FlawFactory(embargoed=False, source=FlawSource.INTERNET)
+        AffectFactory(flaw=flaw1)
+
+        flaw2 = FlawFactory(embargoed=False, source=FlawSource.CUSTOMER)
+        AffectFactory(flaw=flaw2)
+
+        # Test filtering affects by multiple flaw sources with __in suffix
+        response = auth_client().get(
+            f"{test_api_v2_uri}/affects?flaw__source__in=INTERNET,CUSTOMER"
+        )
+        assert response.status_code == status.HTTP_200_OK
+        assert response.json()["count"] == 2
```

Comment:
This test has in total 2 affects and the assert checks for existence of 2 affects, I would add one more affect with different Flaw source to really test whether the filter is applied.